CI builds failing due to feynhiggs download failure

At the moment the CI build is failing when it tries to make feynhiggs due to a download error. This is currently affecting the PRs #373 #371 #369 #366 and #362 (i.e. those that have recently triggered builds)

```
Built target check-rebuild-feynhiggs_2.11.3
Creating directories for 'feynhiggs_2.11.3'
Performing download step for 'feynhiggs_2.11.3'
--2022-08-04 18:47:31--  http://wwwth.mpp.mpg.de/members/heinemey/feynhiggs/newversion/FeynHiggs-2.11.3.tar.gz
Resolving wwwth.mpp.mpg.de (wwwth.mpp.mpg.de)... 134.107.3.76
Connecting to wwwth.mpp.mpg.de (wwwth.mpp.mpg.de)|134.107.3.76|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://wwwth.mpp.mpg.de/members/heinemey/feynhiggs/newversion/FeynHiggs-2.11.3.tar.gz [following]
--2022-08-04 18:47:31--  https://wwwth.mpp.mpg.de/members/heinemey/feynhiggs/newversion/FeynHiggs-2.11.3.tar.gz
Connecting to wwwth.mpp.mpg.de (wwwth.mpp.mpg.de)|134.107.3.76|:443... connected.
ERROR: The certificate of ‘wwwth.mpp.mpg.de’ is not trusted.
ERROR: The certificate of ‘wwwth.mpp.mpg.de’ hasn't got a known issuer.
ERROR: wget failed to download file
Expired or wrong certificate. To download backend/scanner insecurely, use 'IGNORE_HTTP_CERTIFICATE=1 make <backend/scanner>'
make[3]: *** [CMakeFiles/feynhiggs_2.11.3.dir/build.make:105: feynhiggs_2.11.3-prefix/src/feynhiggs_2.11.3-stamp/feynhiggs_2.11.3-download] Error 1
make[2]: *** [CMakeFiles/Makefile2:6665: CMakeFiles/feynhiggs_2.11.3.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:3863: CMakeFiles/feynhiggs.dir/rule] Error 2
make: *** [Makefile:1515: feynhiggs] Error 2
```

This can be fixed for feynhiggs specifically by building with `IGNORE_HTTP_CERTIFICATE=1 make feynhiggs`, but 
1. It's not obvious to me how to adjust the CI to do this
2. Long-term it's not ideal to skip this check

Any suggestions? (happy to make a PR)

--- a/cmake/backends.cmake
+++ b/cmake/backends.cmake
@@ -1175,7 +1175,7 @@ set(FH_CXX_FLAGS "${BACKEND_CXX_FLAGS_NO_BUILD_OPTIMISATIONS}")         #For ski
 check_ditch_status(${name} ${ver} ${dir})
 if(NOT ditched_${name}_${ver})
   ExternalProject_Add(${name}_${ver}
-    DOWNLOAD_COMMAND ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
+    DOWNLOAD_COMMAND IGNORE_HTTP_CERTIFICATE=1 ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
     SOURCE_DIR ${dir}
     BUILD_IN_SOURCE 1
     # Fix bug preventing the use of array bounds checking.
@@ -1203,7 +1203,7 @@ set(FH_CXX_FLAGS "${BACKEND_CXX_FLAGS_NO_BUILD_OPTIMISATIONS}")         #For ski
 check_ditch_status(${name} ${ver} ${dir})
 if(NOT ditched_${name}_${ver})
   ExternalProject_Add(${name}_${ver}
-    DOWNLOAD_COMMAND ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
+    DOWNLOAD_COMMAND IGNORE_HTTP_CERTIFICATE=1 ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
     SOURCE_DIR ${dir}
     BUILD_IN_SOURCE 1
     # Fix bug preventing the use of array bounds checking.
@@ -1232,7 +1232,7 @@ set(FH_CXX_FLAGS "${BACKEND_CXX_FLAGS_NO_BUILD_OPTIMISATIONS}")         #For ski
 check_ditch_status(${name} ${ver} ${dir})
 if(NOT ditched_${name}_${ver})
   ExternalProject_Add(${name}_${ver}
-    DOWNLOAD_COMMAND ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
+    DOWNLOAD_COMMAND IGNORE_HTTP_CERTIFICATE=1 ${DL_BACKEND} ${dl} ${md5} ${dir} ${name} ${ver}
     SOURCE_DIR ${dir}
     BUILD_IN_SOURCE 1
     # Fix bug preventing the use of array bounds checking.

